ocdskingfisherarchive/archive.py

```
@@ -137,7 +163,15 @@ def archive_collection(self, collection):
         self.s3.remove_staging_file(s3_directory + '/metadata.json')
         self.s3.remove_staging_file(s3_directory + '/data.tar.lz4')
 
-        # TODO Delete data files
+        # Delete local files we made
```

I assume the lines before these will raise an exception if they fail? We don't want to reach this code if anything above failed.

---

Yes, there are very few try: blocks and only 3 relevant ones that don't just log and re-raise. Those 3 are very specific and limited (1 is a 404 when getting from S3 and the other 2 are problems parsing dicts in a log file). A discussion for later about opening that up, but to start with I'd just let it crash and we'll examine cases as they arise.